addons/test_mail/tests/test_message_track.py

```diff
@@ -323,8 +323,14 @@ def setUp(self):
         record = self.env['mail.test.track.monetary'].with_user(self.user_employee).with_context(self._test_context).create({
             'company_id': self.user_employee.company_id.id,
         })
+        currency = self.env.ref('base.USD').id
+        record_no_company = self.env['mail.test.track.monetary.no_company'].with_user(self.user_employee).with_context(self._test_context).create({
```

Comment:
Could be create directly in the test that uses it, no need to have it in the global setup I think